README.md

```diff
@@ -6,6 +6,13 @@
 
 ## News
 
+Argo Project is moving from CLA to DCO as a more contributor friendly way to track contributions.
+* https://github.com/apps/dco/
+Please start signing off your contributions by doing ONE of the following:
+* Run `git config commit.gpgsign true` in your argo repo to automatically signoff future commits.
+* Use `git commit -s ...` with each commit to add signoff.
```

Comment:
`error: cannot run gpg: No such file or directory`

Reply:
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/changing-access-permissions-for-wikis

Reply:
Alex, you don't need gpg. You should disable that.

Reply:
when I followed those instructions, it worked